draft-packages/avatar/KaizenDraft/Avatar/styles.module.scss

```diff
@@ -27,26 +27,41 @@
     background: $dt-color-background-color-initials;
   }
 
+  &.company {
+    background: $color-white;
+    border-radius: $border-solid-border-radius;
+  }
+
   // Sizes
 
   &.small {
-    width: $spacing-md;
-    height: $spacing-md;
+    width: $dt-avatar-sm;
+    height: $dt-avatar-sm;
   }
 
   &.medium {
-    width: $spacing-xl;
-    height: $spacing-xl;
+    width: $dt-avatar-md;
+    height: $dt-avatar-md;
   }
 
   &.large {
-    width: $spacing-xxl;
-    height: $spacing-xxl;
+    width: $dt-avatar-lg;
+    height: $dt-avatar-lg;
   }
 
   &.xlarge {
-    width: 7.75rem;
-    height: 7.75rem;
+    width: $dt-avatar-xl;
+    height: $dt-avatar-xl;
+  }
+
+  &.xxlarge {
+    width: $dt-avatar-xxl;
+    height: $dt-avatar-xxl;
+  }
+
+  abbr {
```

Comment:
I'd recommend we add a class to this element and style based on that instead. The reason I say this is because we've run into issues before where we've changed an element's tag and not realised that there's CSS attached to it. Adding a class makes it clear that there's styles attached, and it allows future developers to adjust the tag without needing to change the CSS selector.